blockchain/src/state/read/model.rs

```diff
@@ -200,15 +206,43 @@ pub trait StateQueries {
             return Ok(None);
         };
 
-        let annotator = MoveValueAnnotator::new(resolver);
+        let converter = UmiMoveConverter::new(&resolver);
+        let response = converter.view_resource(&struct_tag, &bytes)?;
+
+        Ok(Some(response))
+    }
+
+    fn table_item_at(
+        &self,
+        handle: &TableHandle,
+        request: TableItemRequest,
+        height: BlockHeight,
+    ) -> Result<Option<MoveValueResponse>, umi_shared::error::Error> {
+        let key_type = request
+            .key_type
+            .try_into()
+            .map_err(|_| UserError::IncorrectTypeLayout)?;
+
+        let value_type = request
+            .value_type
+            .try_into()
+            .map_err(|_| UserError::IncorrectTypeLayout)?;
+
+        let resolver = self.resolver_at(height)?;
+        let converter = UmiMoveConverter::new(&resolver);
+
+        let key = converter.try_into_vm_value(&key_type, request.key)?;
```

Comment:
The key has to be sent in as a string for `u64`. Is that expected?
The test example is for `Table<bool, u64>` and sending key as a bool works in the request, but for `Table<u64, u64>` it expects the key to be a string:
```
"key_type": "u64",
"value_type": "u64",
"key": "2"
```